aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -208,8 +212,49 @@ public void testClientFactorySerialization() throws Exception {
     validateRead(fileIO2);
   }
 
+  @Test
+  public void testDeleteFilesMultipleBatches() throws Exception {
+    S3FileIO s3FileIO = new S3FileIO(clientFactory::s3, getDeletionTestProperties());
+    testDeleteFiles(6, s3FileIO);
```

Comment:
nit, maybe I would think about put it like `deletionBatchSize * 2`

same as below 
- for lessThan, we can do `deletionBatchSize-1`
- for SingleWithRemainer do `deletionBatchSize+1`

so it shall help with understanding and hopefully more robust once `deletionBatchSize` get changed

Reply:
Yeah definitley, this is much better